style.css

```diff
@@ -0,0 +1,398 @@
+* {
```

Comment:
Nice use of the reset CSS. My favorite reset sheet is Eric Meyer's [Reset CSS 2.0](http://cssreset.com/scripts/eric-meyer-reset-css/) which is worth digging into.